### ISSUE TITLE
wrong 'fixed_model_name' parameter name in example restaurantbot

### ISSUE THREAD

**Issue by fabigato** [RasaHQ/rasa_core#177]
**Rasa Core version**: 0.7.9

**Python version**: 3.6

**Operating system** (windows, osx, ...): Ubuntu 16.04

**Issue**: in rasa_core/examples/restaurantbot/bot.py, line 90 uses the wrong parameter 'fixed_model_name'. Should be 'project_name' instead

**Comment by tmbo** [comment]
If you use the command as specified in the makefile to train the NLU model (`python -m rasa_nlu.train -c nlu_model_config.json --fixed_model_name current`) it should just work. Is there an issue with the documentation?

**Comment by fabigato** [comment]
It works when using training the NLU with that command. The problem is if you try to train it by running bot.py with the task=train-nlu argument, as that calls bot.train_nlu() where this line fails:
model_directory = trainer.persist('models/nlu/', fixed_model_name="current")
because it needs the project_name parameter.
This is not a problem if you run it with python -m rasa_nlu.train, as then it executes rasa_nlu.train.do_train(...) which calls the persist function with the right parameters:
persisted_path = trainer.persist(config['path'], persistor,
                                     config['project'],  # <-- here we have the project name
                                     config['fixed_model_name'])
so long story short, there is a problem with the bot.train_nlu() function, that will pass unnoticed if nobody calls that function but uses the python -m rasa_nlu.train command instead

**Comment by Ghostvv** [comment]
project name is set to "default" if you do not specify it

**Comment by tmbo** [comment]
As far as I know the documentation has been fixed to address this.
